configuration_files/map_builder_server.lua

```diff
@@ -0,0 +1,23 @@
+-- Copyright 2017 The Cartographer Authors
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+include "map_builder.lua"
+
+options = {
+  server_address = "0.0.0.0:50051",
+  num_grpc_threads = 4,
+  map_builder = MAP_BUILDER,
+}
+
+return options
```

Comment:
This return prohibits that this file can be included, which is done for the other config files.
It may be better to have (an additional) default config file without return in the future (future PR).